doc/source/getting_started/install.rst

```diff
@@ -255,47 +255,44 @@ For example, :func:`pandas.read_hdf` requires the ``pytables`` package, while
 optional dependency is not installed, pandas will raise an ``ImportError`` when
 the method requiring that dependency is called.
 
+Visualizations and computation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 ========================= ================== =============================================================
 Dependency                Minimum Version    Notes
 ========================= ================== =============================================================
-BeautifulSoup4            4.6.0              HTML parser for read_html (see :ref:`note <optional_html>`)
-Jinja2                    2.10               Conditional formatting with DataFrame.style
-PyQt4                                        Clipboard I/O
-PyQt5                                        Clipboard I/O
-PyTables                  3.5.1              HDF5-based reading / writing
-SQLAlchemy                1.3.0              SQL support for databases other than sqlite
+matplotlib                2.2.3              Plotting library
 SciPy                     1.12.0             Miscellaneous statistical functions
-xlsxwriter                1.0.2              Excel writing
-blosc                     1.17.0             Compression for HDF5
-fsspec                    0.7.4              Handling files aside from local and HTTP
-fastparquet               0.4.0              Parquet reading / writing
-gcsfs                     0.6.0              Google Cloud Storage access
-html5lib                  1.0.1              HTML parser for read_html (see :ref:`note <optional_html>`)
-lxml                      4.3.0              HTML parser for read_html (see :ref:`note <optional_html>`)
-matplotlib                2.2.3              Visualization
+Jinja2                    2.10               Conditional formatting with DataFrame.style
+tabulate                  0.8.7              Printing in Markdown-friendly format (see `tabulate`_)
 numba                     0.46.0             Alternative execution engine for rolling operations
+                                             (see :ref:`Enhancing Performance <enhancingperf.numba>`)
+xarray                    0.12.3             pandas-like API for N-dimensional data
```

Comment:
I would maybe put jinja, tabulate and xarray in a different table, as those are not used in compute anything in pandas (or make plots), but for conversion to / represenation as another format (formatted string, styled html, or xarray object)

Reply:
Yeah... I was basically trying to create a miscellaneous section without directly calling it misc. :sweat_smile: 

I'll just buckle down and label it Miscellaneous unless you have a different suggestion? I just didn't want to split it into two/three items per table.